scripts/azureml-assets/azureml/assets/model/model_utils.py

```diff
@@ -151,3 +153,52 @@ def prepare_model(spec_path, model_config, registry_name, temp_dir):
     except Exception as e:
         logger.log_error(f"prepare model failed for {spec_path}. Error {e}")
         return None, False
+
+
+def update_model_metadata(
+        model_name: str, 
+        model_version: str, 
+        spec_path: Path, 
+        model_config: assets.ModelConfig,
+        registry_name: str, 
+        update: AssetVersionUpdate = None,
+    ):
+    """Update the mutable metadata of already registered Model asset."""
+    try:
+        ml_client = MLClient(credential= AzureCliCredential(), registry_name=registry_name)
+        model = ml_client.models.get(name= model_name, version= model_version)
+
+        if update is not None: ## to update older version of models
```

Comment:
will there be a case when update is not None?

Reply:
Yes,update will be used only when we want to update older versions of model from ADO deploy configs file.
For latest models, updates will be none and we will use only spec files to get tags, description information